test/test_helper.rb

```diff
@@ -7,4 +7,24 @@
 require 'webmock/minitest'
 require 'tmpdir'
 
-Thread.abort_on_exception = true
+def with_env(env)
```

Comment:
these should be in MInitest::Test.class_eval since they otherwise get included into every Object